skipruntime-ts/client/src/client.ts

```diff
@@ -28,6 +28,8 @@ export type {
   Creds,
 };
 
+import WebSocket from "ws";
```

Comment:
The reason I had removed it and relied on the (recent) node implementation available in the global scope, was so that the client could be used in a browser. If we import `ws` (which is node-specific), the client cannot be used directly in the browser anymore.

Reply:
Is it worth it for this specific example?

Reply:
Maybe I'm confused, but I thought this was the implementation of our generic client package, not an example.